package.json

```diff
@@ -1,16 +1,17 @@
 {
     "name": "pact",
-    "displayName": "PACT",
-    "description": "PACT Programming Language",
+    "displayName": "Pact",
+    "description": "Pact Programming Language",
     "icon": "images/kadena_blue_icon.png",
-    "publisher": "Floppie",
```

Comment:
Note that the current extension is published by Floppie in the marketplace with 535 downloads

<img width="604" alt="image" src="https://user-images.githubusercontent.com/8353613/221442031-9c54026a-70c6-4e8a-bbb6-625e4144b9cd.png">